esp/src/src/react/hooks/useWsStore.tsx

```diff
@@ -0,0 +1,19 @@
+//TODO add global, set, localstorage hooks
```

Comment:
Can you open a Jira and remove the comment?

Reply:
https://track.hpccsystems.com/browse/HPCC-23630